modules/monitoring/includes/downtime.php

```diff
@@ -148,8 +149,7 @@ public function get_command_mappings($obj_name, $downtime_window, $comment_prefi
 	 */
 	public function get_command($obj_name, $downtime_window, $comment_prefix = 'AUTO: ') {
 		$command = $this->get_command_mappings($obj_name, $downtime_window, $comment_prefix);
-		$cmd_fmt = 'cmd;obj_name;start;end;is_fixed;0;duration;author;comment';
-		return str_replace(array_keys($command), array_values($command), $cmd_fmt);
+		return implode(';', array_values($command));
```

Comment:
I didn't see anything in the official documentation about array_values guaranteeing the order returned, but I guess you convinced yourself of this? :)

Reply:
Nevermind, was too slow. x)

Reply:
The array is generated directly in `get_command_mappings`, as far as I can tell they are always in that order.